assets/less/app.less

```diff
@@ -1,5 +1,5 @@
 @import 'npm://font-awesome/less/font-awesome';
-@import 'npm://highlightjs/styles/tomorrow';
+@import 'npm://highlight.js/src/styles/tomorrow';
```

Comment:
are we no longer using the previous github style?
if not, we should update the LICENSE.
https://github.com/elixir-lang/ex_doc/blob/master/LICENSE#L49-L56